aep/general/0124/aep.md.j2

```diff
@@ -1,5 +1,86 @@
 # Resource association
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/57) for more
-information.
+APIs sometimes have resource hierarchies that can not be cleanly expressed in
+the usual tree structure. For example, a resource may have a many-to-one
+relationship with two other resource types instead of just one. Alternatively,
+a resource may have a many-to-many relationship with another resource type.
+
+## Guidance
+
+A resource **must** have at most one canonical parent, and `List` requests
+**must not** require two distinct "parents" to work.
+
+### Multiple many-to-one associations
+
+If a resource has a many-to-one relationship with multiple resource types, it
+**must** choose at most one of them to be the canonical parent. The resource
+**may** be associated with other resources through other fields on the
+resource.
+
+{% tab proto %}
+
+{% sample 'multiple_many_to_one.proto', 'message Book' %}
+
+{% tab oas %}
+
+{% sample 'multiple_many_to_one.oas.yaml', 'Book:' %}
+
+{% endtabs %}
+
+When listing resources with multiple associations in this way, the RPC **must**
+treat the `string parent` field as required as discussed in [AEP-132][], and
+**must not** add additional required arguments. The RPC **should** include a
+`string filter` field that allows users to filter by other resource
+associations as discussed in [AEP-160][].
+
+**Note:** Resource reference fields **must** accept the same resource name
+format that is used in the `name` field of the referenced resource.
```

Comment:
```suggestion
**Note:** Resource reference fields **must** accept the [resource path](/resource-path) of the referenced resource.
```

We call name "path" in AEPs, rather than "name". Also I feel like this guidance isn't a little redundant - we should just link to /resource-path.